docs/plugin_developers_guide/01_Fledge_plugins.rst

```diff
@@ -41,11 +41,136 @@ Plugins
+| timestamp  | string           | Fledge timestamp when this asset was generated                                                        |
++------------+------------------+-------------------------------------------------------------------------------------------------------+
+| key        | string           | unique ID                                                                                             |
++------------+------------------+-------------------------------------------------------------------------------------------------------+
+| readings   | dict             | {"<name-1>": <value-1> [, "<name-2>": <value-2>,... ["user_ts": <device-timestamp>]]}                 |
++------------+------------------+-------------------------------------------------------------------------------------------------------+
+
+Plugin configuration
+====================
+
+Different plugin types have required configuration entries.
+
+Required configuration entries include:
+
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| Entry                  | Required by            | Description          | Fields                                                |
++========================+========================+======================+=======================================================+
+| plugin                 | all                    | Basic plugin name    | description: "<Describe plugin>" |br|                 |
+|                        |                        | and information      | type: "string" |br|                                   |
+|                        |                        |                      | name: "<plugin name>" |br|                            |
+|                        |                        |                      | readonly: "true"                                      |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| enable                 | Filter |br|            | Enable/disable       | description: "<Describe plugin being enabled>" |br|   |
+|                        | Notification |br|      | plugin function      | type: "boolean"                                |br|   |
+|                        | Notification delivery  |                      | default: "false"                               |br|   |
+|                        |                        |                      | displayName: "<name for UI display>"           |br|   |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| asset                  | South |br|             | Asset where data     | description: "<name of asset being monitored>" |br|   |
```

Comment:
There is nothing mandatory about having an asset configuration item in the south.